src/main/java/pro/beam/api/resource/chat/methods/WhisperMethod.java

```diff
@@ -0,0 +1,46 @@
+package pro.beam.api.resource.chat.methods;
+
+import com.google.common.collect.ImmutableList;
+import pro.beam.api.resource.BeamUser;
+import pro.beam.api.resource.chat.AbstractChatMethod;
+
+import java.util.List;
+import java.util.Random;
+
+public class WhisperMethod extends AbstractChatMethod {
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        protected BeamUser to;
```

Comment:
I'm going to leave this as-is for now because I think the type of the field provides accurate enough information about its intention. It's also an protected field in a Builder class, so I'm not too concerned about it :smile: